src/styles/blocks/card.scss

```
@@ -0,0 +1,120 @@
+.card {
+  background-color: #fff;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    width: 160px ;
+    height: 134px ;
+    margin: 0 auto;
+    margin-bottom: 40px;
+
+  }
+
+  &__name {
+    font-weight: 500;
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover .card__name {
+    color: #34568b;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-weight: 400;
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__buy {
+    width: 166px;
```

just the full parent size
```suggestion
    width: 100%;
```